server/logger.go

```diff
@@ -0,0 +1,55 @@
+package server
```

Comment:
Neat! Maybe it's time for the SDK to provide a `logging` package, wdyt? So how about moving this into its own `logger` (or `log`, or `logging`) package in the repository's root directory?

Reply:
Why?

Reply:
Just a personal preference, I was imagining the user doing something like:

```
import "github.com/cosmos/cosmos-sdk/logwrapper"

...

     logwrapper.New(...)
```


But it's just a personal preference. This is great as is already!

Reply:
I think that's overkill IMHO.